src/main/java/com/easypost/model/ReferralCustomerCollection.java

```diff
@@ -2,43 +2,10 @@
 
 import java.util.List;
 
+import lombok.Getter;
+
+@Getter
 public final class ReferralCustomerCollection extends EasyPostResource {
     private List<ReferralCustomer> referralCustomers;
-    private boolean hasMore;
-
-    /**
-     * Get a list of ReferralCustomers.
-     *
-     * @return List of ReferralCustomers objects
-     */
-    public List<ReferralCustomer> getReferralCustomers() {
-        return referralCustomers;
-    }
-
-    /**
-     * Set a list of ReferralCustomers.
-     *
-     * @param referralCustomers List of ReferralCustomers objects
-     */
-    public void setReferralCustomers(final List<ReferralCustomer> referralCustomers) {
-        this.referralCustomers = referralCustomers;
-    }
-
-    /**
-     * Get whether there are more ReferralCustomers to retrieve.
-     *
-     * @return whether there are more ReferralCustomers to retrieve
-     */
-    public boolean getHasMore() {
-        return hasMore;
-    }
-
-    /**
-     * Set whether there are more ReferralCustomers to retrieve.
-     *
-     * @param hasMore Boolean whether there are more ReferralCustomers to retrieve
-     */
-    public void setHasMore(final boolean hasMore) {
-        this.hasMore = hasMore;
-    }
+    private Boolean hasMore;
```

Comment:
Question: We meant to change this from `boolean` to `Boolean`?

Reply:
Yes, this is the only `boolean` type of the `hasMore` across our Java source code. I will list this out in the changelog too since it's a minor breaking change.

Reply:
So wait, all our booleans in this library are technically `Boolean`?

Reply:
Yes, all attributes are wrapper classes, because we have to account for `null` in the return response.

Reply:
🙃